client/gutenberg/extensions/giphy/editor.scss

```diff
@@ -44,3 +44,8 @@
 		}
 	}
 }
+.components-panel__body-giphy-branding {
+	svg path {
```

Comment:
Could we add `svg { display: block; margin: 0 auto; max-width: 200px }` so it doesn't look huge on small screens and stays centred

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/30351/commits/4c7b7fd2193eb13cb6d6f04123d9efdbcab43065